web/src/test/java/org/springframework/security/web/csrf/CookieCsrfTokenRepositoryTests.java

```diff
@@ -300,23 +331,42 @@ public void loadDeferredTokenWhenExistsThenLoaded() {
 	}
 
 	@Test
-	public void setCookieNameNullIllegalArgumentException() {
+	void cookieInitializer() {
+		String domainName = "example.com";
+		String customPath = "/custompath";
+		String sameSitePolicy = "Strict";
+		this.repository.addCookieInitializer(builder -> builder.domain(domainName));
```

Comment:
I believe the project's formatter will require `builder` to be in parentheses. Please run `./gradlew format check` to be sure.

Reply:
Thanks for the tip. I double-checked it, and it is not required.